lib/let_me/policy.ex

```diff
@@ -1022,6 +1029,18 @@ defmodule LetMe.Policy do
       MyApp.Policy.authorize!(:article_view, %{age: 10})
       # => true
 
+  You can achieve the same functionality dynamically using the opts on `authorize!`:
+
+      object :article do
+        action :view do
+          pre_hooks {MyApp.Policy.Checks, :set_age}
+          allow min_age: 50
+        end
+      end
+
+      MyApp.Policy.authorize!(:article_view, %{age: 10}, 50)
```

Comment:
The object argument is missing here, and the last argument needs to be an enumerable.